src/details/ArborX_DetailsMinimumSpanningTree.hpp

```diff
@@ -15,14 +15,14 @@
 #include <ArborX_DetailsAlgorithms.hpp>
 #include <ArborX_DetailsHappyTreeFriends.hpp>
 #include <ArborX_DetailsKokkosExtArithmeticTraits.hpp>
-#include <ArborX_DetailsKokkosExtMinMaxOperations.hpp>
 #include <ArborX_DetailsKokkosExtSwap.hpp>
 #include <ArborX_DetailsUtils.hpp>
 #include <ArborX_DetailsWeightedEdge.hpp>
 
 #include <Kokkos_BitManipulation.hpp>
 #include <Kokkos_Core.hpp>
 #include <Kokkos_MathematicalFunctions.hpp> // isfinite, signbit
+#include <Kokkos_MinMax.hpp>
```

Comment:
```suggestion
```

Reply:
Also means this PR conflicts with #1139 
Not sure about opening both concurrently.  Would have been better to either merge or do one after the other,